src/lib.rs

```diff
@@ -83,13 +83,15 @@
 //! ```
 #![forbid(unsafe_code, missing_docs)]
 
+pub mod client;
```

Comment:
Should this be a public module?

Reply:
It depends on how you wish to expose the sub-clients. They are `crate::client::SpecificClient` right now. If `mod client` is not public, you can not click through to their documentation:

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/174509/112691335-89303600-8e53-11eb-9453-f367a476eed1.png">

Reply:
Right fair enough! Lets leave this the way it is then